### ISSUE TITLE
Add storybook for component library

### ISSUE THREAD

**Issue by YajanaRao** [YajanaRao/Serenity#132]
Need to add storybook for the component library which is inside `packages/components`. Components are already there, we need a simple solution to explore them via storybook.

**Comment by ankit060801** [comment]
sir please assign me this this!!

**Comment by YajanaRao** [comment]
Sure @ankit060801 . Let me know if you need anything ?

**Comment by YajanaRao** [comment]
@ankit060801 How is this going ?. Any progress ?

**Comment by ankit060801** [comment]
sir please assign to someone else.i am not able to do

**Comment by sarthak7awasthi** [comment]
Hi, I am a react developer. I'm familiar with storybook. Can I get assigned to this issue? I'll keep you in loop regarding the progress.

**Comment by YajanaRao** [comment]
@sarthak7awasthi I don't think this issue is relevant anymore but If you like to work on anything in specific feel free to open a issue with changes you would like to do and create a PR for it.
